src/app/pages/home/home.component.html

```diff
@@ -18,8 +18,8 @@ <h5 class="card-title text-uppercase text-muted mb-0">Our Professionals</h5>
                 </div>
               </div>
               <p class="mt-3 mb-0 text-muted text-sm">
-                <span class="text-success mr-2"><i class="fa fa-arrow-up"></i> 3.48%</span>
-                <span class="text-nowrap">Since last month</span>
+                <span class="text-success mr-2"><i class="fa fa-arrow-up"></i> </span>
```

Comment:
Arrows needs to be taken out as well. You are supposed to clear the all but the report only

Reply:
copy that!